src/FSharp.AWS.DynamoDB/Expression/UpdateExpr.fs

```diff
@@ -295,7 +295,9 @@ let extractUpdateOps (exprs: IntermediateUpdateExprs) =
         | SpecificCall2 <@ List.append @> (None, _, _, [ left; right ]) ->
             UpdateValue.EList_Append (extractOperand pickler left) (extractOperand pickler right)
 
-        | SpecificCall2 <@ defaultArg @> (None, _, _, [ AttributeGet attr; operand ]) ->
+        | SpecificCall2 <@ defaultArg @> (None, _, _, [ AttributeGet attr; operand ])
+        | Some'(SpecificCall2 <@ defaultArg @> (None, _, _, [ AttributeGet attr; operand ]))
```

Comment:
this looks very cryptic - maybe `NewSome` or `OptionSome` or `SomeCaseCtor` naming instead?